pkg/sources/fake/fake.go

```diff
@@ -25,10 +46,87 @@ func getCAN(id uint16) msgs.CAN {
 
 // GenFake generates fake data
 func GenFake(bus *pubsub.MessageBus) {
```

Comment:
tbh felt kind of dirty writing it, but since it was purely to generate fake data, I didn't put too much thought into it. I'll create a ticket so it gets tracked (ELEC-137).